settings.gradle

```diff
@@ -44,6 +44,12 @@ dependencyResolutionManagement {
     }
 }
 rootProject.name = 'logging-log4j-samples'
+def log4jVersion = providers.environmentVariable("LOG4J_VERSION")
```

Comment:
Do we provide this environment variable anywhere, or is it just for convenience?